transport/outbound.go

```diff
@@ -22,9 +22,6 @@ package transport
 
 import "golang.org/x/net/context"
 
-// Outbounds is a collection of Outbound transports.
```

Comment:
Why not keep `Outbounds` as an alias for `map[string]Outbound` so that the initialization looks cleaner, e.g.
```go
Outbounds: Outbounds{
  "moe": http.Outbound("..."),
}
```
vs the current code
```go
Outbounds: map[string]transport.Outbound{
  "moe": http.Outbound(...),
},
```